### ISSUE TITLE
Review demo site before sharing with Caitlin

### ISSUE THREAD

**Issue by dbridenbeck** [dbridenbeck/whidbey-herbal#102]

**Comment by dbridenbeck** [comment]
TODO: 
1. Store Locator Map overflows right screen between laptop -> tablet and in tablet resolution
2. Remove SOLD OUT warning on Hero Img in Product page

**Comment by dbridenbeck** [comment]
DONE: 
5. **FeaturedPhotos & PhotoSection need height: auto**: This required removing flexbox from their parent containers and using inline-block for images instead.

**Comment by dbridenbeck** [comment]
DONE: 
- **Scroll to top page transition isn't working on mobile**: This was a considerable task - it resulted in me removing React-Spring from the project and instead just using React-Router's ScrollToTop component to scroll to top. However, on mobile this wasn't working on the home page which was due to many elements exceeding 100vw, which caused the Header to not stay fixed. Through exploring options to this I decided to simplify the code base by removing Page-Wrapper, as this was only necessary for React-Spring to work correctly. Then it was easier to re-style the components in Home.js to not exceed 100vw.
